packages/eslint-plugin-turbo/lib/constants.ts

```diff
@@ -0,0 +1,5 @@
+const RULES = {
+  noUncachedEnvVars: `no-uncached-env-vars`,
```

Comment:
Not a strong opinion, but I don't love uncached.
```suggestion
  noUnenumeratedEnvVars: `no-unenumerated-env-vars`,
```

Reply:
How about `no-unenumerated-env-vars-dependencies` ? 

(double negative with `no-unenumerated..` is also not unbad, but i know eslint has a pattern there!)

Reply:
Settles on `no-undeclared-env-vars`